e2e/helperActions/addProductToCart.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, Page } from '@playwright/test';
+import { closeNotification } from './notification';
+
+export const addProductToCart = async (page: Page) => {
```

Comment:
I propose to immediately remove the helpers for tests that will definitely not be used